pages/api/nurses/[id].js

```diff
@@ -3,13 +3,23 @@ import Nurse from "@/db/models/Nurse";
 
 export default async function handler(request, response) {
   await dbConnect();
+  const { id } = request.query;
 
   if (request.method === "GET") {
-    const nurse = await Nurse.findById(request.query.id);
+    const nurse = await Nurse.findById(id);
 
     if (!nurse) {
-      return response.status(404).json({ status: "You have no team 😩!" });
+      return response.status(404).json({ status: "Team not Found 🫣" });
     }
+
     response.status(200).json(nurse);
   }
+
+  if (request.method === "PUT") {
+    await Nurse.findByIdAndUpdate(id, {
+      $set: request.body,
+    });
+
+    response.status(200).json({ message: "Yes!!! a New Nurse in Team! 🍻" });
```

Comment:
Since the PUT is updating, it's not a new nurse ^^
I would send the updated entity back as a response though, and not a message.